### DISCUSSION TITLE
ERROR [auto download 3/8] RUN . /clone.sh stable-diffusion-stability-ai https://github.com/Stability-AI/stablediffusion.git cf1d67a6fd5ea1aa600c4df

### DISCUSSION THREAD

**Discussion by grgomer** [AbdBarho/stable-diffusion-webui-docker#655] [Q&A]
=> ERROR [auto download 3/8] RUN . /clone.sh stable-diffusion-stability-ai https://github.com/Stability-AI/stablediffusion.git cf1d67a6fd5ea1aa600c4df  189.4s
------
 > [auto download 3/8] RUN . /clone.sh stable-diffusion-stability-ai https://github.com/Stability-AI/stablediffusion.git cf1d67a6fd5ea1aa600c4df58e5b47da45f6bdbf   && rm -rf assets data/**/*.png data/**/*.jpg data/**/*.gif:
0.183 + mkdir -p /repositories/stable-diffusion-stability-ai
0.184 + cd /repositories/stable-diffusion-stability-ai
0.184 + git init
0.185 hint: Using 'master' as the name for the initial branch. This default branch name
0.185 hint: is subject to change. To configure the initial branch name to use in all
0.185 hint: of your new repositories, which will suppress this warning, call:
0.185 hint:
0.185 hint:     git config --global init.defaultBranch <name>
0.185 hint:
0.185 hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
0.185 hint: 'development'. The just-created branch can be renamed via this command:
0.185 hint:
0.185 hint:     git branch -m <name>
0.185 Initialized empty Git repository in /repositories/stable-diffusion-stability-ai/.git/
0.185 + git remote add origin https://github.com/Stability-AI/stablediffusion.git
0.186 + git fetch origin cf1d67a6fd5ea1aa600c4df58e5b47da45f6bdbf '--depth=1'
188.6 error: RPC failed; curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8)
188.6 error: 2997 bytes of body are still expected
188.6 fetch-pack: unexpected disconnect while reading sideband packet
188.6 fatal: early EOF
188.6 fatal: fetch-pack: invalid index-pack output
------
failed to solve: process "/bin/sh -c . /clone.sh stable-diffusion-stability-ai https://github.com/Stability-AI/stablediffusion.git cf1d67a6fd5ea1aa600c4df58e5b47da45f6bdbf   && rm -rf assets data/**/*.png data/**/*.jpg data/**/*.gif" did not complete successfully: exit code: 128

Could you please help me about that error?

**Comment by InAUGral** [comment]
I get the same issue, did you ever find a fix? Trying to install this via Docker running in a Ubuntu VM in Proxmox.
